sabnzbd/misc.py

```diff
@@ -1307,6 +1307,9 @@ def get_base_url(url):
     url_host = urlparse(url).hostname
     if url_host:
         url_split = url_host.split(".")
+        # Exception for localhost and IPv6 addresses
```

Comment:
Because Google's Favicon service will only work on root domains (so nzbgeek.com, but nog api.nzbgeek.com) I updated the function a while ago to extra the root-domain instead of just the domain:
```return ".".join(len(url_split[-2]) < 4 and url_split[-3:] or url_split[-2:])```

But that is based on the assumption that there will be at least three ```.```, if there's less it breaks and gives a 500 page. So when using ```localhost``` or ```[::ffff]``` (or any IPv6).
This fixes that and just returns the whole domain instead of trying to find the root-domain.